week1/My-own-modified/index.js

```diff
@@ -0,0 +1,39 @@
+"use strict";
+const http = require("http");
+const server = http.createServer((req, res) => {
+  console.log("Creating Server");
+});
+let state = 10;
+let original = 10;
+server.on("request", (req, res) => {
+  function responseWrite(val) {
```

Comment:
I really like your use of a helper function, but in this code here, you're re-declaring this function on every request - no need to do that. You could move this outside the `server.on` block.